src/olympia/addons/templates/addons/replacement_addons.html

```diff
@@ -0,0 +1 @@
+{% extends "impala/base_shared.html" %}
```

Comment:
Some good candidates for the placeholder here: http://textfiles.com/underconstruction/